src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import moment from 'moment'
+
+export default () => (
+  <footer role='main'>
+    <div className='container'>
+      <p>© {moment().format('YYYY')} Streamr</p>
+      <p>Designed and built by Tyler Petresky, Steven Petryk, Greg White, and Maxcell Wilson.</p>
```

Comment:
@Tresky @GregKWhite @maxcell any requests for attribution links? Twitters, websites, etc?